packages/moonstone/Slider/Slider.js

```diff
@@ -254,7 +254,8 @@ const SliderBase = kind({
 			activateOnFocus,
 			active
 		}),
-		tooltip: ({tooltip}) => tooltip === true ? ProgressBarTooltip : tooltip
+		tooltip: ({tooltip}) => tooltip === true ? ProgressBarTooltip : tooltip,
+		value: ({min, value}) => value || min
```

Comment:
This will force a zero value to be `min`.  We have to check against `value == null`.  Can someone whip up a patch quick-like?

Reply:
Ah, good call. Looks like setting a default of `min` for `value` (which I did improperly above) would work without us having to type-check. I've updated the logic.